config/initializers/core_extensions/devise/strategies/guest.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Devise
+  module Strategies
+    # strategy for a user guest
+    class Guest < Base
```

Comment:
Please add a comment that this sets the current user to an instance of the class `Object`